.circleci/config.yml

```diff
@@ -161,6 +161,15 @@ prebuild-job: &prebuild-job
         - master
         - /v\d+\.\d+/
 
+deply-reliability:
```

Comment:
```suggestion
deploy-reliability:
```

Reply:
deeply reliable

Reply:
fixed in latest push

Reply:
Looks like the typo was in multiple places.

Reply:
Does it? I can't find `deply` anywhere else.